mandatory/2-function-creation.js

```diff
@@ -5,7 +5,12 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  for (i = 0; i < strArr.length; i++) {
```

Comment:
Nice!
Could we rewrite this function using `map` or `forEach`? How would that look? :)